src/sql/strong-oracle/drop.sql

```diff
@@ -0,0 +1,154 @@
+
+DROP PUBLIC SYNONYM N_NOTIFICATIONS FORCE;
+/
+
+DROP PUBLIC SYNONYM N_LU_CASE_STATUS FORCE;
+/
+
+DROP PUBLIC SYNONYM N_CASES FORCE;
+/
+
+DROP PUBLIC SYNONYM N_LU_NOTIFICATION_STATUS FORCE;
+/
+
+DROP SEQUENCE seq_n_cases_pk;
+
+DROP SEQUENCE seq_n_notifications_pk;
+/
+
+ALTER TABLE N_NOTIFICATIONS
+DROP CONSTRAINT N_FK_CASES_NOTIFICATIONS__CASE_I;
+/
+
+ALTER TABLE N_NOTIFICATIONS
+DROP CONSTRAINT N_FK_LU_NOTIFICATION_STATE_NOTIF;
+/
+
+ALTER TABLE N_CASES
+DROP CONSTRAINT N_FK_LU_CAST_STATE_CASES__STATUS;
+/
+
+ALTER TABLE N_NOTIFICATIONS
+DROP PRIMARY KEY CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_LU_NOTIFICATION_STATUS
+DROP CONSTRAINT N_XAK1LU_NOTIFICATION_STATUS CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_LU_NOTIFICATION_STATUS
+DROP PRIMARY KEY CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_CASES
+DROP PRIMARY KEY CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_CASES
+DROP CONSTRAINT N_XAK1CASES_GUID CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_LU_CASE_STATUS
+DROP PRIMARY KEY CASCADE  DROP INDEX;
+/
+
+ALTER TABLE N_LU_CASE_STATUS
+DROP CONSTRAINT N_XAK1_LU_CASE_STATE CASCADE  DROP INDEX;
+/
+
+
+DROP TRIGGER trg_N_NOTIFICATIONS_update
+;
+/
+
+DROP TRIGGER trg_N_NOTIFICATIONS_insert
+;
+/
+
+DROP TRIGGER trg_N_CASES_update
+;/
+
+DROP TRIGGER trg_N_CASES_insert
+;
+/
+
+
+DROP INDEX N_XPK_NOTIFICATIONS;
+/
+
+DROP INDEX N_XIF1_NOTIFICATIONS;
+/
+
+DROP INDEX N_XIF2_NOTIFICATIONS;
+/
+
+DROP TRIGGER tD_notifications;
+/
+
+DROP TRIGGER tU_notifications;
+/
+
+DROP TRIGGER tI_notifications;
+/
+
+DROP TABLE N_NOTIFICATIONS CASCADE CONSTRAINTS PURGE;
+/
+
+DROP INDEX N_XAK1LU_NOTIFICATION_STATUS;
+/
+
+DROP INDEX XPKLU_NOTIFICATION_STATUS;
+/
+
+DROP TRIGGER tD_lu_notification_state;
+/
+
+DROP TRIGGER tU_lu_notification_state;
+/
+
+DROP TRIGGER tI_lu_notification_state;
+/
+
+DROP TABLE N_LU_NOTIFICATION_STATUS CASCADE CONSTRAINTS PURGE;
+/
+
+DROP INDEX XPKN_CASES;
+/
+
+DROP INDEX XAK1CASES_GUID;
+/
+
+DROP INDEX XIF1N_CASES;
+/
+
+DROP TRIGGER tD_cases;
+/
+
+DROP TRIGGER tU_cases;
+/
+
+DROP TRIGGER tI_cases;
```

Comment:
these should be removed because you altered the name of the create triggers to something else and in the drop you followed that name
these are the old names that I originally proposed